README.md

```diff
@@ -105,6 +105,23 @@ True
 True
 ```
 
+> [!NOTE]
+>
+> Since `v0.14.1`, a new namespace `optree.pytree` is introduced as aliases for `optree.tree_*` functions. The following examples are equivalent to the above:
+>
+> ```python
+> import optree.pytree as pt
+> >>> tree = {'b': (2, [3, 4]), 'a': 1, 'c': 5, 'd': 6}
+> >>> pt.flatten(tree)
+> ([1, 2, 3, 4, 5, 6], PyTreeSpec({'a': *, 'b': (*, [*, *]), 'c': *, 'd': *}))
+> >>> pt.flatten(1)
+> ([1], PyTreeSpec(*))
+> >>> pt.flatten(None)
+> ([], PyTreeSpec(None))
```

Comment:
I'd like to add an example for `tree_map`. It is the key motivation that we add this shortcut module.

```suggestion
> ([], PyTreeSpec(None))
> >>> pt.map(lambda x: x**2, tree)
> {'b': (4, [9, 16]), 'a': 1, 'c': 25, 'd': 36}
```